### PR TITLE
Add support for select and download in overdue search screen

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.shortcut.com/simpledotorg/story/8657/add-support-for-selected-download-and-share-in-overdue-search-screen

### CODE REVIEW

**Comment by janhavisinghh** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4096** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/4096" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/4096?utm_source=stack-comment).

**Review by msasikanth — APPROVED** [review]

**Comment by janhavisinghh** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/4096?utm_source=stack-comment).


Job Reference: ccty8I4tbtUfa9Qoe6j0

**Pull request merged**
